content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -106,6 +106,7 @@ This example uses the default provisioner for Minikube. Please update the follow
 
 {{< codenew file="application/cassandra/cassandra-statefulset.yaml" >}}
 
+1. Review the StatefulSet yaml.  Specifically, review the Storage Class and confirm/adjust as appropriate.  More information on persistent volumes at https://kubernetes.io/docs/concepts/storage/persistent-volumes/.  Note the provisioner and type are specific for a given cloud provider.
```

Comment:
How about some of the changes from this rewording:
```suggestion
1. Review the StatefulSet manifest. If you want to use this example beyond a basic tutorial and would like the storage to persist, review the StorageClass for the and confirm or adjust as appropriate. [Persistent Volumes](/docs/concepts/storage/persistent-volumes/) explains more about the storage concepts involved here. You need to pick a provisioner and type that is valid for your cluster; for example, use a provisioner for your cloud provider.
```

?

Reply:
Keep in mind that the example won't as is, unless you happen to be using minikube.  People may be deploying on aws, gcloud, etc...and will need to update the storage for this to work.  Is that clear?

Reply:
Also, your comment "want" storage to persist is off I think.  StatefulSet requires a pv from what I understand.

Reply:
I don't know the best way to word it, so I hope these comments help, and happy to review another draft.

Reply:
Hello @gerlt . Thanks for this pull request.
Because there is an existing note before the manifest, what do think about adjusting the note with some of your new content? I'll let you work out the details.
Note text:
This example uses the default provisioner for Minikube. You must update the `casandra-statefulset.yaml` manifest with the `provisioner` and `type` for your given cloud provider.

Could this replace line 110 (added some text from @sftim)?
```
1.Review the StorageClass settings in `casandra-statefulset.yaml` and modify as appropriate. 
   [Persistent Volumes](/docs/concepts/storage/persistent-volumes/) explains more about the storage concepts involved here.
```

Reply:
Yes, that should work.